bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -117,6 +119,10 @@ <h2 class="heading-tertiary">
 
       <button class="try-hello" id="try-hello-intro">{{ _('Try Firefox Hello') }}</button>
 
+      <aside id="powered-by">
```

Comment:
Does this need to be in an l10n tag, or do we already have these strings translated?